WultraMobileTokenSDK/Operations/Utils/WMTTOTPUtils.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2023 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Utility class used for handling TOTP
+public class WMTTOTPUtils {
+    
+    /// Data payload which is
+    public struct WMTOperationOTPData: Codable {
```

Comment:
The name is wrong, it should be TOTP, not OTP, anyway, the whole prefix is unnecessary.

The result is `WMTTOTPUtils.WMTOperationOTPData` which is too complex. The struct name could be just `Data` that will result in `WMTTOTPUtils.Data` or lift the struct outside of the utils class and name it `WMTOperationTOTPData`.